<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupDialog</class>
 <widget class="QDialog" name="SetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Connection Type</string>
   </property>
   <widget class="QComboBox" name="ConnectionTypecomboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>271</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Serial Settings</string>
   </property>
   <widget class="QComboBox" name="ComPortcomboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="BaudcomboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="DataBitscomboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="ParitycomboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="StopBitcomboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Com. Mode</string>
   </property>
   <widget class="QRadioButton" name="ASCIIradioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ASCII</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RTUradioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RTU</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Timeout</string>
   </property>
   <widget class="QLineEdit" name="TimeoutlineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>[ms]</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelpushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
